client_opts.go

```diff
@@ -178,3 +178,11 @@ func WithImageHandler(h images.Handler) RemoteOpt {
 		return nil
 	}
 }
+
+// WithDecryptionKeys sets private keys in client struct
+func WithDecryptionKeys(decryptionKeys []string) RemoteOpt {
+	return func(client *Client, c *RemoteContext) error {
+		client.decryptionKeys = decryptionKeys
```

Comment:
Are these the whole keys are identifiers?

Reply:
Sorry, I do not completely understand the question. The keys are passed in in their raw form, so either in pem or der format or whatever format they may be plus a potential password.